src/main/kotlin/net/atos/zac/app/signalering/SignaleringRestService.kt

```diff
@@ -80,29 +71,41 @@ class SignaleringRestService @Inject constructor(
         }
     }
 
+    /**
+     * Lists zaken signaleringen for the given signaleringsType.
+     */
+    @GET
+    @Path("/zaken/{type}")
+    fun listZakenSignaleringen(
+        @PathParam("type") signaleringsType: SignaleringType.Type,
+        @QueryParam("pageNumber") pageNumber: Int,
+        @QueryParam("pageSize") pageSize: Int,
+    ): List<RestZaakOverzicht> =
+        signaleringService.listZakenSignaleringenPage(signaleringsType, pageNumber, pageSize)
+
+    /**
+     * !!! defaults are to be removed when we switch to paging
+     */
     @GET
     @Path("/taken/{type}")
     fun listTakenSignaleringen(
-        @PathParam("type") signaleringsType: SignaleringType.Type
+        @PathParam("type") signaleringsType: SignaleringType.Type,
+        @QueryParam("pageNumber") pageNumber: Int = 0,
+        @QueryParam("pageSize") pageSize: Int = 1000
```

Comment:
In acc crit it states that number of entries shwon should be 5;  would it make all faster if we set this default to 5?

Reply:
This will break the current UI. So we'll have non-working TEST until paging is implemented.

When FE is ready I anyway need to do a cleanup, so will change this default to 5 (or will remove it)